ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/ClusterDetailsTable.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { useState } from 'react';
+import { generatePath, Link } from 'react-router-dom';
+import { Button, ButtonVariant, Text, TextVariants, Tooltip } from '@patternfly/react-core';
+import { Table, Tbody, Td, Th, Thead, Tr } from '@patternfly/react-table';
+
+import IconText from 'Components/PatternFly/IconText/IconText';
+import TbodyUnified from 'Components/TableStateTemplates/TbodyUnified';
+import { ComplianceCheckResult } from 'services/ComplianceResultsService';
+import { TableUIState } from 'utils/getTableUIState';
+
+import { coverageCheckDetailsPath } from './compliance.coverage.routes';
+import { getClusterResultsStatusObject } from './compliance.coverage.utils';
+import CheckStatusModal from './components/CheckStatusModal';
+
+export type ClusterDetailsTableProps = {
+    clusterId: string;
+    profileName: string;
+    tableState: TableUIState<ComplianceCheckResult>;
+};
+
+function ClusterDetailsTable({ clusterId, profileName, tableState }: ClusterDetailsTableProps) {
+    const [selectedCheckResult, setSelectedCheckResult] = useState<ComplianceCheckResult | null>(
+        null
+    );
+    return (
+        <>
+            <Table>
+                <Thead>
+                    <Tr>
+                        <Th>Check</Th>
+                        <Td modifier="fitContent" width={10}>
+                            Controls
+                        </Td>
+                        <Th modifier="fitContent" width={10}>
+                            Compliance status
+                        </Th>
+                    </Tr>
+                </Thead>
+                <TbodyUnified
+                    tableState={tableState}
+                    colSpan={3}
+                    errorProps={{
+                        title: 'There was an error loading results for this cluster',
+                    }}
+                    emptyProps={{
+                        message: 'No results found for this cluster',
+                    }}
+                    filteredEmptyProps={{
+                        title: 'No results found',
+                        message: 'Clear all filters and try again',
+                    }}
+                    renderer={({ data }) => (
+                        <Tbody>
+                            {data.map((checkResult) => {
+                                const { checkName, rationale, status } = checkResult;
+                                const clusterStatusObject = getClusterResultsStatusObject(status);
+
+                                return (
+                                    <Tr key={clusterId}>
```

Comment:
Is it `key={checkName}` here?

Reply:
Ah, thought I fixed that one as well. Updated